File: server/dotnet/sdk-dotnet-example/Presentation/Mappers/HostedTokenizationMapper.cs
using Business.DTOs.HostedTokenizations;
using Presentation.Models.GetHostedTokenizations;

namespace Presentation.Mappers;

public static class HostedTokenizationMapper
{
    public static GetHostedTokenizationResponse Map(GetHostedTokenizationResponseDto response)
    {
        return new()
        {
            HostedTokenizationId = response.HostedTokenizationId,
            HostedTokenizationUrl = response.HostedTokenizationUrl
        };
    }
}

File: server/nodejs/src/presentation/mappers/hosted-tokenization-mapper.ts
import { GetHostedTokenizationResponse } from '../models/hosted-tokenization/get-hosted-tokenization-response';
import { GetHostedTokenizationResponseDto } from '../../business/dtos/hosted-tokenization/get-hosted-tokenization-response-dto';

export const HostedTokenizationMapper = {
    fromDto: (dto: GetHostedTokenizationResponseDto): GetHostedTokenizationResponse => {
        return {
            hostedTokenizationId: dto.hostedTokenizationId,
            hostedTokenizationUrl: dto.hostedTokenizationUrl,
        };
    },
};
